common/common.scss

```diff
@@ -46,7 +46,7 @@
         }
       }
 
-      .d-icon-lock {
+      .d-icon-lock, .d-icon-unlock, .d-icon-unlock-alt, .d-icon-lock-open, .d-icon-user-secret, .d-icon-user-lock {
```

Comment:
Rather than targeting all the variations, we can apply a class to the "lock icon" and target that class instead.
```js
contents.push(iconNode(lockIcon, { class: "category-lock-icon" }));
```


```scss
.category-lock-icon {
...
}
```